src/openforms/utils/decorators.py

```diff
@@ -65,3 +68,43 @@ def _wrapped_view_func(request, *args, **kwargs):
         return _wrapped_view_func
 
     return decorator
+
+
+def dbfields_exception_handler(
+    exceptions: tuple[Type[Exception]],
+):
+    """
+    A decorator that wraps a try catch around the formfields_for_dbfields function.
+    If the decorator catches an exception it will return a base formfield with
+    an error message discribing that an error has occured.
```

Comment:
```suggestion
    A decorator that wraps a try catch around the formfields_for_dbfields function.
    If the decorator catches an exception it will return a base formfield with
    an error message describing that an error has occurred.
```